zeekpkg/_util.py

```diff
@@ -278,14 +282,30 @@ def git_remote_urls(repo):
 
 def is_sha1(s):
     if not s:
-        return False;
+        return False
 
     if len(s) != 40:
         return False
 
     for c in s:
-        if c not in {'a', 'b', 'c', 'd', 'e', 'f',
-                     '0', '1', '2', '3', '4', '5', '6', '7', '8', '9'}:
+        if c not in {
```

Comment:
Brrr... Maybe replace with a single return at the end of this function as follows or regex it?
```
hexdigits = set(string.hexdigits.lower())
return all([c in hexdigits for c in s])
```

Reply:
While I completely agree that this can be improved, this line appears in the diff due to automated edits. Let's keep non-automated changes out of this PR as much as possible. I'd rather drop this PR completely than address every issue in code touched by it.

Reply:
Sure, I do agree with that sentiment. This hunk was just standing out so prominently I left the comment. It's the only one I intended to point out. Please don't close the PR before merging it.

Reply:
Okay, pushed a patch fixing _just this one instance_ 😉

Reply:
Elegant fix!